WcaOnRails/db/migrate/20231122153755_create_wfc_dues_redirects.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateWfcDuesRedirects < ActiveRecord::Migration[7.0]
+  def change
+    create_table :wfc_dues_redirects do |t|
+      t.string :redirect_type, null: false
```

Comment:
You can drop this because the polymorphic `references` below adds a `redirect_source_type` column under the hood